src/spac/spatial_analysis.py

```diff
@@ -90,23 +91,51 @@ def spatial_interaction(
         - :class:`tuple` - prune the final graph to only contain
             edges in interval `[min(radius), max(radius)]`.
 
+    cmap : str, default 'seismic'
+        The colormap to use for the plot. The 'seismic' color map
+        consist of three color regions: red for positive, blue for negative,
+        and the white at the center. This color map effectively represents
+        the nature of the spatial interaction analysis results,
+        where positive values indicate clustering and
+        negative values indicate seperation. For more color maps, please visit
+        https://matplotlib.org/stable/tutorials/colors/colormaps.html
+
     **kwargs
         Keyword arguments for matplotlib.pyplot.text()
-    Returns:
-    -------
-    ax_dictionary : dictionary
-        The returned dictionary containse matplotlib.axes.Axes
-        under 'Ax' key and optional matrix under 'Matrix' key.
-        If stratify is used, the function will return dictionary
-        of Axes and optional matrix with keys representing
-        the stratification groups.
-        For example: if stratify is not used and matrix is called,
-        the matrix can be acquired by result['Matrix'], and axes
-        from result['Ax']. If stratify is used and has two levels,
-        "A" and "B", the axes for A can be extracted by
-        result['Ax']['A'] and matrix through result['Matrix']['A'].
-
 
+    Returns
+    -------
+    dict
+        A dictionary containing the results of the spatial interaction
+        analysis. The keys of the dictionary depend on the parameters
+        passed to the function:
+
+        Ax : dict or matplotlib.axes.Axes
+            If `stratify_by` is not used, returns a single
+            matplotlib.axes.Axes object. If `stratify_by` is used,
+            returns a dictionary of Axes objects, with keys
+            representing the stratification groups.
+
+        Matrix : dict or tuple, optional
+            Returned only if `return_matrix=True`. If `stratify_by` is
+            not used, returns:
+            - For "Neighborhood Enrichment": a tuple with the z-score
+              matrix and enrichment counts matrix.
+            - For "Cluster Interaction Matrix": the interaction
+              matrix. If `stratify_by` is used, returns a dictionary

```

Comment:
@ruiheesi, line 120 mentions that stratify_bis is not used, but this line says it is used. Does the general condition that stratify_by is not used applies to this line 125?